index.bs

```diff
@@ -922,6 +931,12 @@ optionally evidence of [=user consent=] to a specific transaction.
         authenticators are eligible to participate in a {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}
         operation. See [[#attachment]] for a description of the attachment values and their meanings.
 
+    :   <dfn>requireResidentKey</dfn>
+    ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=credential private keys=]. If the
+        parameter is set to <a>true</a>, the authenticator SHOULD create a [=resident key=] when creating a [=scoped credential=].
```

Comment:
This ties into #410 then: if the authenticator model section didn't exist, and merely referred to CTAP, it'd be clear that we need to look there to see the normative effect of `requireResidentKey`.

You should still remove the normative statements from the descriptions of these fields, and just write factual statements about how the CTAP algorithms use their values. (e.g. "the authenticator creates ...") Otherwise, when implementers are reading the algorithm definitions to figure out what to implement, they're likely to miss the extra requirements we place in the field descriptions, leading to bugs.